BraintreeCore/src/main/java/com/braintreepayments/api/BraintreeRequestCodes.java

```diff
@@ -26,4 +27,5 @@
     int GOOGLE_PAY = 13593;
     int SAMSUNG_PAY = 13595;
     int LOCAL_PAYMENT = 13596;
+    int SEPA = 13597;
```

Comment:
We just incremented the previous request code, but is there any info on how these numbers were determined or what this new one should be?